package.json

```diff
@@ -26,6 +26,11 @@
     "webpack-dev-server": "^3.11.0"
   },
   "scripts": {
-    "lint": "NODE_ENV=development eslint --fix app/javascript/**"
+    "lint": "NODE_ENV=development eslint --fix app/javascript/**",
+    "bootstrap": "docker compose run dev script/setup",
+    "docker": "docker compose run dev",
+    "dev": "docker compose run --service-ports app",
+    "test": "docker compose run tester bundle exec rake",
+    "test:prepare": "docker compose run tester script/prepare_tests"
```

Comment:
please dont add tasks that are not strictly for JS. this is not a JS repo and people would be confused. simply add a script or look into the existing entrypoints directory for inspiration.

Reply:
So you want bash scripts in `scripts` instead?

Reply:
Please give me an example of such a script in the codebase and point me to the folder which should be used to store such scripts.

Out of curiosity: Node developers are used to run scripts like: `yarn test` or `bun test` and not bash scripts. Does ruby / rails not provide a way to run such scripts (including bash commands) with something like `bundle exec test`?